### PR TITLE
Change IRC references to Libera.Chat

### PR DESCRIPTION
**Pull request by amCap1712**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ If you are interested in helping out, consider
 
 If you are interested in contributing code or documentation,
 please have a look at the [issue tracker](https://tickets.metabrainz.org/browse/LB)
-or come visit us in the #metabrainz IRC channel on irc.freenode.net.
+or come visit us in the #metabrainz IRC channel on irc.libera.net.
```
Comment (shivam-kapila) [inline README.md:45]:
```suggestion
or come visit us in the #metabrainz IRC channel on irc.libera.chat.
```

File: listenbrainz/webserver/templates/index/contribute.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h3>Developers</h3>
   </p>
   <p>
      Have a look at the <a href="https://github.com/metabrainz/listenbrainz-server">GitHub repository</a> for this
-     project to get started. You may also consider heading to our IRC channel #metabrainz on irc.freenode.net and
+     project to get started. You may also consider heading to our IRC channel #metabrainz on irc.libera.net and
```
Comment (shivam-kapila) [inline listenbrainz/webserver/templates/index/contribute.html:20]:
```suggestion
     project to get started. You may also consider heading to our IRC channel #metabrainz on irc.libera.chat and
```

**Review by shivam-kapila — CHANGES_REQUESTED** [review]

**Comment by amCap1712** [comment]
@shivam-kapila Thanks!

**Review by alastair — APPROVED** [review]
Looks good. Do we also have a template repository that contains this information?

**Comment by amCap1712** [comment]
@alastair Yes, it has already been updated. https://github.com/metabrainz/.github/pull/2

**Pull request merged**